client/finality-grandpa/src/authorities.rs

```diff
@@ -461,8 +471,7 @@ where
 				);
 
 				telemetry!(
-					CONSENSUS_INFO;
-					"afg.applying_forced_authority_set_change";
+					telemetry; CONSENSUS_INFO; "afg.applying_forced_authority_set_change";
```

Comment:
These are still parameters and should be put onto a new line for each of them

Reply:
I like the format as it is now: always write the macro in multi line, keep the first 3 parameters on their own line, the rest on every line. I guess it's a matter a taste I don't mind changing it. Maybe the macro syntax could be improved to be more clear.

Reply:
What about this syntax?

```rust
				telemetry!(
					(self.telemetry, CONSENSUS_INFO, "afg.voter_command_change_authorities") => {
						"number" => ?new.canon_number,
						"hash" => ?new.canon_hash,
						"voters" => ?voters,
						"set_id" => ?new.set_id,
					}
				);
```

Reply:
The new syntax is just more writing, not any improvement. Just pot everything on a new line.

Reply:
Done in c7139fd057

Reply:
And 993040b3f3